app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -124,15 +142,19 @@ describe('components/ConnectedSidebar', () => {
     wrapper.unmount();
   });
 
-  it('shows the correct logg', () => {
+  it('correctly expand admin-form-config menu', () => {
```

Comment:
@ashfahan I thought we were to remove this test cases since its already catered for in  `Test order of menu'` test case?

Reply:
these expansions tests were originally written by @HaseemUlHaq @haseemVD i just replicate them over other sidebar to like fill the gap

@HaseemUlHaq can you guide us better on this ?